internal/uvm/create.go

```diff
@@ -188,12 +191,8 @@ func (uvm *UtilityVM) Close() (err error) {
 		uvm.hcsSystem.Terminate(ctx)
 		uvm.Wait()
 	}
-	if uvm.gc != nil {
-		uvm.gc.Close()
-	}
-	if uvm.gcListener != nil {
-		uvm.gcListener.Close()
-	}
+
+	uvm.CloseGCSConnection()
```

Comment:
We don't care about the error here? Should we at least log it?

Reply:
Added a log.